docs/Documentation.md

```diff
@@ -1524,17 +1524,17 @@ With a convention file layout:
 And extension.kt code:
 ```kotlin
 class MySourceProcessor : SourceProcessorExtension {
-    val replace : StringParameter
+    val replace: StringParameter
     val with: StringParameter
-    
-    override fun process(input : SourceInput, output: SourceOutput) {
+
+    override fun process(input: SourceInput, output: SourceOutput) {
         val replaced = input.readText().replaceAll(replace, with)
         output.writeText(replaced)
     }
 }
 ```
 
-The Amper engine would be able to quickly discover the DSL schema for `setting:my-source-processor:` when evaluatig the project structure. And also compiler and execute arbitrary login defined in Kotlin file.     
+The Amper engine would be able to quickly discover the DSL schema for `setting:my-source-processor:` when evaluating the project structure. And also compile and execute arbitrary login defined in Kotlin file.     
```

Comment:
I'm not sure if this sentence is correct:
> And also compile and execute arbitrary login defined in Kotlin file.

But I've not changed it as I don't understand what it should look like.

Reply:
I believe `login` is a typo for `logic` here

Reply:
Should I create a separate PR for this fix, since this one is closed?
If so, maybe I should also update [lines about the supported Gradle version](https://github.com/JetBrains/amper/commit/a76712073f9acab2d02d46bf9b1ba260e4bf546a). Taking into account [your commit](https://github.com/JetBrains/amper/commit/7de48b123ff3c7280c16fb8b6bacc817faab5584) adding support for Gradle 8.7+

Reply:
FYI I made the changes in the main branch, so no need for a PR